src/commands/cloud-run/flare.ts

```diff
@@ -0,0 +1,254 @@
+import IService = google.cloud.run.v2.IService
+import fs from 'fs'
+import path from 'path'
+import process from 'process'
+import util from 'util'
+
+import {ServicesClient} from '@google-cloud/run'
+import {google} from '@google-cloud/run/build/protos/protos'
+import chalk from 'chalk'
+import {Command} from 'clipanion'
+import {GoogleAuth} from 'google-auth-library'
+
+import {API_KEY_ENV_VAR, CI_API_KEY_ENV_VAR, FLARE_OUTPUT_DIRECTORY, FLARE_ZIP_FILE_NAME} from '../../constants'
+import {createDirectories, deleteFolder, writeFile, zipContents} from '../../helpers/fileSystem'
+import {sendToDatadog} from '../../helpers/flareFunctions'
+import {requestConfirmation} from '../../helpers/prompt'
+import * as helpersRenderer from '../../helpers/renderer'
+
+import {maskEnvVar} from '../lambda/functions/commons'
+
+const SERVICE_CONFIG_FILE_NAME = 'service_config.json'
+
+export class CloudRunFlareCommand extends Command {
+  private isDryRun = false
+  private service?: string
+  private project?: string
+  private location?: string
+  private caseId?: string
+  private email?: string
+  private apiKey?: string
+
+  /**
+   * Entry point for the `cloud-run flare` command.
+   * Gathers Cloud Run service configuration and sends it to Datadog.
+   * @returns 0 if the command ran successfully, 1 otherwise.
+   */
+  public async execute() {
+    this.context.stdout.write(helpersRenderer.renderFlareHeader('Cloud Run', this.isDryRun))
+
+    const errorMessages: string[] = []
+    // Validate service
+    if (this.service === undefined) {
+      errorMessages.push(helpersRenderer.renderError('No service specified. [-s,--service]'))
+    }
+
+    // Validate project
+    if (this.project === undefined) {
+      errorMessages.push(helpersRenderer.renderError('No project specified. [-p,--project]'))
+    }
+
+    // Validate location
+    if (this.location === undefined) {
+      errorMessages.push(helpersRenderer.renderError('No location specified. [-l,--location]'))
+    }
```

Comment:
Lambda uses the `AWS_DEFAULT_REGION` environment variable. However, GCP doesn't use environment variables. Instead, it uses 'properties' stored in file called `gcloud`: https://cloud.google.com/sdk/docs/properties

I looked around the Datadog docs and couldn't find an existing environment variable similar to `AWS_DEFAULT_REGION` for GCP. Is there one we could use that I can't find for some reason?